ntbs-service/Pages/Notifications/Edit/Patient.cshtml.cs

```diff
@@ -86,6 +94,18 @@ private void UpdatePatientFlags()
             }
         }
 
+
+        public async Task<ContentResult> OnGetValidatePostcode(string postcode, bool shouldValidateFull)
+        {   
+            var foundPostcode = await PostcodeService.FindPostcode(postcode);
+            var patientDetails = new PatientDetails() {
+                PostcodeToLookup = foundPostcode?.Postcode,
+                ShouldValidateFull = shouldValidateFull
+            };
+
+            return validationService.ValidateProperty(patientDetails, "Postcode", postcode);
```

Comment:
Now that we have the generic methods, shouldn't we just write this as
`return validationService.ValidateModelProperty<PatientDetails>("Postcode", foundPostcode?.Postcode, shouldValidateFull);`?

Reply:
the problem here is that we need to specify two fields Postcode and PostcodeToLookup,
and then postcode is validated based on itself and postcodetolookup value

Reply:
ah sorry I missed that! There is a generic ValidateMultipleProperties on validationservice though which should be usable here?